utils/enrichToken.test.ts

```diff
@@ -0,0 +1,46 @@
+import { getEnrichedToken } from './enrichToken';
```

Comment:
Calling out that for this file, I put it in alongside the util file (in the same directory).  For our query tests, those are located in a specific tests folder, but I think it's generally easier to find the test if you locate it with the file it's testing.  It's a bit weird to have two different patterns, but any thoughts on this?

We could actually adopt this pattern by splitting our resolver code into separate ts files in a specific directory, and putting the query tests along them (but that's a nice to have for another day).